cmd/snap-confine/snap-confine.caps

```diff
@@ -1 +1 @@
-cap_chown,cap_dac_override,cap_dac_read_search,cap_fowner,cap_sys_chroot,cap_sys_ptrace,cap_sys_admin=p
+cap_chown,cap_dac_override,cap_dac_read_search,cap_fowner,cap_setgid,cap_setuid,cap_sys_chroot,cap_sys_ptrace,cap_sys_admin=p
```

Comment:
maybe we should have 2 files instead of just one, or drop additional caps from this file and only append setgid/setuid specifically when packaging for relevant systems (ubuntu/debian/snap)?

Reply:
fair enough, but since we're going to squash merge, could use update arch PKGBUILD to remove cap_setgid and cap_setuid from the caps file during build?

not sure, but maybe worth to add a TODO in snapcraft.yaml and relevant debian/rules files with note to remove this in the future.

Reply:
We now have two files